lib/ModelMixins/TableMixin.ts

```diff
@@ -194,12 +195,15 @@ export default function TableMixin<T extends Constructor<Model<TableTraits>>>(
             points.push({ x, y });
           }
 
+          const colorId = `color-${this.name}-${yColumn.name}`;
           const chartData = new ChartData({
             name: yColumn.name,
             categoryName: this.name,
             points,
             units: yColumn.traits.units,
-            color: line.color
+            getColor: () => {
+              return line.color || getNextChartColor(colorId);
+            }
```

Comment:
Does this need to be evaluated later?

Reply:
I think yes. For instance we use a fixed color (white) to show chart items in feature info panel, eagerly allocating a color in this case will use up the pool.